docs/mesos/DockerizedDeployment.md

```diff
@@ -17,7 +17,9 @@ In these instructions, we will run all cluster services as [Docker][docker] cont
  * Calico
  * Marathon (Mesos framework)
 
-In these brief instructions, I'll concentrate on getting Mesos and Calico up and running as quickly as possible.  This means leaving out the details of how to configure highly-available services.  Instead, I'll install Zookeeper, etcd, and the Mesos Master on the same "master" node.
+We'll concentrate on getting Mesos and Calico up and running as quickly as possible.  This means leaving out the details of how to configure highly-available services.  Instead, we'll install Zookeeper, etcd, and the Mesos Master on the same "master" node.
+
+For an automated install of a two-host Dockerized Mesos cluster using Vagrant and VirtualBox, follow the [Vagrant Dockerized Mesos Guide](DockerizedVagrant.md).
```

Comment:
Can you boldface this whole sentence? I think it would draw better  attention to it